hail_scripts/utils/clinvar.py

```diff
@@ -21,9 +40,68 @@
         "practice_guideline": 4,
     }
 )
+CLINVAR_PATHOGENICITIES_LOOKUP = hl.dict(
+    hl.enumerate(
+        # NB: sorted by pathogenicity
+        [
+            'Pathogenic',
+            'Pathogenic/Likely_pathogenic',
+            'Pathogenic/Likely_pathogenic/Likely_risk_allele',
+            'Pathogenic/Likely_risk_allele',
+            'Likely_pathogenic',
+            'Likely_pathogenic/Likely_risk_allele',
+            'Likely_risk_allele',
+            'Conflicting_interpretations_of_pathogenicity',
+            'Established_risk_allele',
+            'Uncertain_risk_allele',
+            'Uncertain_significance/Uncertain_risk_allele',
+            'Uncertain_significance',
+            'Likely_benign',
+            'Benign/Likely_benign',
+            'Benign',
+            CLINVAR_DEFAULT_PATHOGENICITY,
+        ],
+        index_first=False
+    )
+)
 
-
-def download_and_import_latest_clinvar_vcf(genome_version: str) -> hl.MatrixTable:
+def parsed_clnsig(ht: hl.Table):
+    return (
+        hl.delimit(ht.info.CLNSIG)
+        .replace(
+            'Likely_pathogenic,_low_penetrance', 'Likely_pathogenic|low_penetrance',
+        )
+        .replace(
+            '/Pathogenic,_low_penetrance', '|low_penetrance',
+        )
+        .split(r'\|')
+    )
+
+def parsed_clnsigconf(ht: hl.Table):
+
+    def parse_to_count(entry: str):
+        splt = entry.split(r'\(') # pattern, count = entry... if destructuring worked on a hail expression!
+        return hl.tuple([
+            splt[0], 
+            hl.int32(splt[1][:-1])
+        ])
+
+    return (
+        hl.delimit(ht.info.CLNSIGCONF)
+        .replace(',_low_penetrance', '')
+        .split(r'\|')
+        .map(parse_to_count)
+        .group_by(lambda x: x[0])
+        .map_values(lambda values: (
```

Comment:
Shouldn't each pathogenicity only appear once with a count? Why do we need to do a group and summation?

Reply:
See the test here for an example: https://github.com/broadinstitute/seqr-loading-pipelines/pull/430/files/515135c47a496173421f57e277caf2ac4a8d7bc2#diff-9a66d5d4eb152375e3619fb8871357ab5a5196c837b9a0e9d03d4f57bdf6a66eR38

`Pathogenic,_low_penetrance` shows up as a unique pathogenicity.

Reply:
😢 unfortunately I can't think of a better way to do this, given that thats how the data is delivered

Reply:
I tried thinking of a different approach as well but couldnt